README.md

```diff
@@ -24,6 +24,37 @@ wget --header='Accept: application/vnd.github.v3.raw' -O container_update.sh htt
 4. Customize the default config according to your specific requirements *(see below)*
 5. Create a cron job for this script *(after testing 🫠)*
 
+### Docker
+1. Clone this repo and go to folder.
+   ```
+   git clone https://github.com/Keonik1/Docker-Container-Updater.git
```

Comment:
Shouldn't we change the URL to `https://github.com/jansppenrade2/Docker-Container-Updater.git`?